js/index.js

```diff
@@ -40,3 +40,78 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Navigation
+let navbarLinks = document.querySelector('nav').children;
+for(let i = 0; i < navbarLinks.length; i++){
+  navbarLinks[i].innerText = siteContent.nav["nav-item-" + (i + 1)];
+  navbarLinks[i].style.color = 'green';
+}
+
+// Adding new links to navigation
+let navbar = document.querySelector('nav');
+
+// Prepend
+let firstLink = document.createElement('a');
+firstLink.innerText = 'Home';
+firstLink.style.color = 'green';
+firstLink.href = '#';
+navbar.prepend(firstLink);
+
+// Append
+let lastLink = document.createElement('a');
+lastLink.innerText = 'Extras';
+lastLink.style.color = 'green';
+lastLink.href = '#';
+navbar.appendChild(lastLink);
+
+// Call To Action
+let ctaText = document.querySelector('div.cta-text').children;
+// CTA Header
+ctaText[0].innerText = siteContent.cta['h1'];
+// CTA Button
+ctaText[1].innerText = siteContent.cta['button'];
+// CTA Image
+let ctaImg = document.getElementById('cta-img');
```

Comment:
Generally, if you not going to mess with a variable any further (e.g. _You're not going to reassign `ctaImg` to be something else_), it's probably a good idea to use `const`. It will _help_ prevent accidental changes to the variable, and it will communicate to developers that the variable isn't meant to be changed.